v-next/hardhat-ignition/src/internal/tasks/track-tx.ts

```diff
@@ -0,0 +1,54 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types/hre";
+import type { NewTaskActionFunction } from "hardhat/types/tasks";
+
+import path from "node:path";
+
+import { trackTransaction } from "@nomicfoundation/ignition-core";
+
+interface TrackTxArguments {
+  txHash: string;
+  deploymentId: string;
+}
+
+const taskTransactions: NewTaskActionFunction<TrackTxArguments> = async (
+  { txHash, deploymentId },
+  hre: HardhatRuntimeEnvironment,
+) => {
+  const deploymentDir = path.join(
+    hre.config.paths.ignition,
+    "deployments",
+    deploymentId,
+  );
+
+  const connection = await hre.network.connect();
+
+  let output: string | void;
+  try {
+    output = await trackTransaction(
+      deploymentDir,
+      txHash,
+      connection.provider,
+      hre.config.ignition.requiredConfirmations,
+    );
+  } catch (e) {
```

Comment:
Should this now just be removed based on the previous pattern?

Reply:
Yep! Good catch